### PR TITLE
Updated deployer.md.

### PR DESCRIPTION
**Pull request by dbialek2**
Edited so that the content is more understandable.
@lalyos @matyix

### CODE REVIEW

File: docs/aws/deployer.md [hunk 1]
```diff
@@ -29,70 +29,70 @@ First initialize `cbd` by creating a `Profile` file:
...
 **AWS Account Keys**
 
-There are 2 ways to create AWS credentials in Cloudbreak.
+There are two ways to create AWS credentials in Cloudbreak:
 
-* Key-based: It requires your AWS access and secret key and Cloudbreak will use this key to launch the resources. This key needs to be provided when you create your credential in Cloudbreak either with Cloudbreak UI or Cloudbreak CLI.
-* Role-based: It requires a valid IAM User role and Cloudbreak will assume this role to get a temporary access and secret key. For this action you need to set your AWS key in the `Profile` file.
-We suggest to use the keys of a valid **IAM User** here.
+* **Key-based:** This requires your AWS access key and secret key pair. Cloudbreak will use these keys to launch the resources. Provide the keys when you create your credentials in Cloudbreak either with Cloudbreak UI or Cloudbreak CLI.
+* **Role-based:** This requires a valid IAM User role (We suggest that you use keys of a valid **IAM User**). Cloudbreak will assume this role to get a temporary access and secret key pair. To set your AWS keys in the `Profile` file, add these lines:
```
Comment (lalyos) [inline docs/aws/deployer.md]: the term "IAM User role" is confusing, it just an "AWS role". (i know it was already there, but lets fix it)

the `(We suggest that you use keys of a valid **IAM User**)` part is in the same sentence where we try to explain the "role-based". Maybe we should include a small explanaition, that the acces/secret keys which have to go into the Profile, needs only an IAM User with only "AssumeRole" policy.

**Comment by lalyos** [comment]
I think this document needs structural changes. Mandatory config/steps should be grouped together, and be on the top of the page, while all optional config/steps should be moved to an "optional" section to the bottom.

- for the latest AWS images the "Initialize your Profile" step is not needed anymore (i will check it for both centos/amazon linux version)

- "Custom Tags" should be moved to the end and marked as optional.

- key-based / role-based: As this page describes the process to get cloudbreak up and running, maybe something like below is more clear:
  - You can use acceskey/secretkey credentials later in the UI to provision a DP cluster, in that case, you dont need eny extra configuration.

For the role based, actually there again 2 sub options:
- Start the instance, with an "AssumeRole" policy. Maybe even put a link to "AWS Instance Profiles" explanation": http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html
 or to http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
- Other option is to set the acceskey/secretkey (of an IAM user with an "AssumeRole" policy) into Profile.

But this last one is the most confusing, probably we should push them to the first option:
If ypo want to use Role base credentials, start your instance with an Instance Profile.

and describe how thats done.

**Comment by dbialek2** [comment]
I agree with your comments. I'm implementing your comments @lalyos

**Comment by dbialek2** [comment]
@lalyos I just realized that the info about IAM roles is included in the next section of Cloudbreak docs (Provisioning Prerequisites > Set up IAM role.) at https://github.com/sequenceiq/cloudbreak-docs/blob/master/docs/aws/iamrole.md 
 
Do you think it would make sense to add the info that we were just editing to this section and delete it from deployer.md? This is what I propose since some of the info is the same.

Let me know!

**Pull request merged**